cerberus/custom_checks/custom_checks.py

```diff
@@ -90,6 +91,26 @@ def check_PV():
         return False
 
 
+def check_kyverno():
+    logging.info("Check if the Kyverno ConfigMap is okay.")
+    result = False
+
+    create_config = runcommand.invoke(
+        'oc create -n kyverno configmap simple-test --from-literal=data="asdf" && echo "successed"')
+    time.sleep(5)
```

Comment:
Do we need to sleep for 5 seconds? I think `oc create` wont return until the object is created, preventing any race conditions.

Reply:
You are right, I found that in py, it waits for each line to complete before moving to the next.